config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.